### DISCUSSION TITLE
Improvement to github issues section

### DISCUSSION THREAD

**Discussion by shubhamkumarpanday** [community/community#22077] [New to GitHub]
<p>hello github team<br>
can  you please replace p tag with pre tag in issues section of github repo.<br>
currently It is problematic to share python code in github issues. it is removing indentation while displaying</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You can use <a href="https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks">Markdown code blocks</a> to post code in your issues and comments, optionally even with syntax highlighting. :slightly_smiling_face:</p>

**Comment by shubhamkumarpanday** [comment]
<p>Thankyou <a class="mention" href="/airtower-luna">@airtower-luna</a> for the help :slight_smile: It’s working…</p>
